k8s/demo/common/sscs/sscs-bulk-scan.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/sscs/bulk-scan:pr-1048-bcdd1e7-20211005094911 #{"$imagepolicy": "flux-system:demo-sscs-bulk-scan"}
+      image: hmctspublic.azurecr.io/sscs/bulk-scan:pr-1072-2646f34-20211008185359 #{"$imagepolicy": "flux-system:demo-sscs-bulk-scan"}
```

Comment:
you don't need to update this yourself the automation will do it, but I guess saves you a little time.